packages/vite/src/node/config.ts

```diff
@@ -294,13 +299,16 @@ export async function resolveConfig(
     config.root ? path.resolve(config.root) : process.cwd()
   )
 
+  const clientAlias = [
+    { find: /^[\/]?@vite\/env/, replacement: () => ENV_ENTRY },
```

Comment:
When browser request `@vite\env`, the `@rollup/plugin-alias` will get `/@vite/evn`. However, when `vite:import-analysis` parses @vite/client, it will get `@vite/evn`.